Samples/SwiftKitSampleApp/src/main/java/com/example/swift/HelloJava2Swift.java

```diff
@@ -78,6 +82,20 @@ static void examples() {
             });
         }
 
+        // Example of using 'Data'.
+        try (var arena = AllocatingSwiftArena.ofConfined()) {
+            var origBytes = arena.allocateFrom("foobar");
+            var origDat = Data.init(origBytes, origBytes.byteSize(), arena);
+            SwiftRuntime.trace("origDat.count = " + origDat.getCount());
+            
+            var retDat = MySwiftLibrary.globalReceiveReturnData(origDat, arena);
+            retDat.withUnsafeBytes((retBytes) -> {
+                var str = retBytes.getString(0);
+                SwiftRuntime.trace("retStr=" + str);
+            });
+        }
```

Comment:
Very nice! 

btw I'm thinking we should start trying to move all these examples into tests, so we'll get a better report "what failed" when CI fails.

But this we can do later separately